Rubberduck.Core/UI/Refactorings/MoveFolder/MoveMultipleFoldersView.xaml.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Rubberduck.Refactorings;
+using Rubberduck.Refactorings.MoveFolder;
+
+namespace Rubberduck.UI.Refactorings.MoveFolder
+{
+    public partial class MoveMultipleFoldersView : IRefactoringView<MoveMultipleFoldersModel>
+    {
+        public MoveMultipleFoldersView()
+        {
+            InitializeComponent();
+
+            Loaded += (o, e) =>
+                {
+                    MoveToFolderTextBox.Focus();
+                    MoveToFolderTextBox.SelectAll();
+                };
```

Comment:
How does the inline EventHandler delegate get de-registered?

Reply:
I have to admit that I simply copied this from the `Rename` dialog. Maybe, whoever wrote that can answer.

Reply:
I think it simply gets unregistered when the view gets GCed. Remember, it is the event that keeps the handler alive, not the other way around.

I will add explicit de-registration, nonetheless.